myBike0.1

OleksiiBielovolov opened the pull request:
[DEMO LINK](https://OleksiiBielovolov.github.io/layout_miami/)

OleksiiBielovolov commented:
I finished the header, I'm waiting for your corrections to improve the code.

SanyaBratashchuk reviewed (changes requested):
Good job. Keep working. Re-request review when you finish whole landing

OleksiiBielovolov commented:
Completely completed the landing and modernized it a little. Added transform to menu__link, bikes__photo and the-details__photo also added animation to icon--phone. I'm sorry that I didn't delete the photo that I wanted to use in this landing page.

SanyaBratashchuk reviewed (changes requested):
1. Add transition for images hover effects 
2. All external links should be open in the separate tab 
3. Make all fields required
4. Form submit shouldn't lead user to top of the page 
5. Can't scroll menu in landscape orientation 
![image](https://user-images.githubusercontent.com/97547824/222242946-21baff27-703b-4ced-8d45-72e98d6ae212.png)

DarkMistyRoom reviewed (changes requested):
Good job! Let's improve some points:
- the logo should link to the header
![image](https://user-images.githubusercontent.com/98764641/222684584-0ecbaf78-0ad5-48df-ae7e-1ac43fa48d30.png)

- fix this gap
![image](https://user-images.githubusercontent.com/98764641/222684849-383c663d-74c1-491e-a2a0-7fc07aec844e.png)
![image](https://user-images.githubusercontent.com/98764641/222684886-c5046af4-df0b-4f3d-905e-835513ff81ba.png)

- the number also should be clickable, but when I try to move the cursor on it, it disappears
![image](https://user-images.githubusercontent.com/98764641/222685049-afb24b5e-90a7-4f5c-8b0b-135fe8e4a65e.png)

- the cursor becomes 'zoom' over these images, but the user actually can't zoom them. better make the simple arrow to avoid misunderstanding
![image](https://user-images.githubusercontent.com/98764641/222685433-a2a6affd-54e1-421a-b9fb-c86a66e38e5e.png)

- fix the font on the textarea
![image](https://user-images.githubusercontent.com/98764641/222686061-1bacd35f-892d-4586-a75d-475c007d36dd.png)

- you have double padding on the tab
![image](https://user-images.githubusercontent.com/98764641/222686436-dec20422-ac32-4a0a-9d4a-394e9eeb4b45.png)

- this indent is 24px on the mockup
![image](https://user-images.githubusercontent.com/98764641/222686777-82c9bd44-807b-4982-b4a5-325671dee043.png)
![image](https://user-images.githubusercontent.com/98764641/222686837-ac6b50a2-c379-43c4-ace4-2a9f86043bf5.png)

- I noticed it. It's cool!
![image](https://user-images.githubusercontent.com/98764641/222686976-33431541-6949-4e43-a59b-9302a0f6d112.png)

loralevitska reviewed (changes requested):
Almost done)

- Add padding to burger menu for more clickable zone
![image](https://user-images.githubusercontent.com/70009205/223183686-0dc8d7c8-fb5b-4a33-b233-5d037f9bffdd.png)
- Remove target_blank for anchors 
![image](https://user-images.githubusercontent.com/70009205/223189640-869eef0f-7495-49ff-a141-d8ffa0b48aed.png)
- Fix placeholder font-style 
![image](https://user-images.githubusercontent.com/70009205/223190078-1e719526-e142-42c1-9fbc-b3adc20c18d4.png)
- Remove autocomplete background 
![image](https://user-images.githubusercontent.com/70009205/223190349-70808f1c-29ce-4287-9bd8-0c6b2148d1a7.png)
